src/core/Akka.Remote.Tests/RemoteRouterSpec.cs

```diff
@@ -110,7 +110,7 @@ public void RemoteRouter_must_deploy_its_children_on_remote_host_driven_by_confi
             for (var i = 0; i < 5; i++)
             {
                 router.Tell("", probe.Ref);
-                var expected = probe.ExpectMsg<ActorRef>();
+                var expected = probe.ExpectMsg<ActorRef>(TimeSpan.FromSeconds(5));
```

Comment:
That's a good idea - I'll modify this commit and do that.

Looks like the Inbox and Resizer specs need some love too.